.woodpecker/.verify-pr.yml

```diff
@@ -0,0 +1,19 @@
+pipeline:
+  install:
+    image: danlynn/ember-cli:4.7.0-node_16.17
```

Comment:
Not the same ember-cli version, but that's the only one with node 16 and ember-cli will use the version from the node_modules folder anyways.